library/tract-renderer/src/main/res/layout/tract_content_image.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<org.cru.godtools.tract.widget.TractPicassoImageView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<org.ccci.gto.android.common.picasso.view.SimplePicassoImageView xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/image"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:scaleType="centerCrop"
```

Comment:
when wrapping content, `scaleType="centerCrop"` will resize the image to fill the view, but not crop anything.